src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -35,9 +44,10 @@ public override IDisplayResult Display(ListPart listPart, BuildPartDisplayContex
                     {
                         var pager = await GetPagerSlimAsync(context);
                         var settings = context.TypePartDefinition.GetSettings<ListPartSettings>();
-
+                        var containeditemOptions = new ContainedItemOptions();
+                        Models.ContentsStatus published = (Models.ContentsStatus)2;
```

Comment:
Ennek nincs értelme.

Reply:
Az eredetiben, mikor még publishedOnly volt, a felső metódusban true, az alsóban false érték volt beírva. Ezért akartam, hogy felül a published, alul a None értéket adjuk át, és állítottam ezt a változót a published-re átadás előtt. 
Hogy lehetne ezt szebben?

Reply:
Értelmezd kérlek, mi történik itt. Egy integert literalt castolsz egy enumra. Miért nem használod az enumot közvetlenül? Miért nem használodaz enum értékét közvetlenül a lenti metódushívásban?

Reply:
Ah igaz! Működik, csak annyit néztem el, hogy a paraméterbe is Models.ContentStatus-al kell

Reply:
Javítva sima ContentsStatus.Published -re, valamiért először a VS azt javasolta Models-el írjam